incskill_site/templates/login.html

```diff
@@ -118,9 +156,83 @@ <h1 id="title">Login</h1>
     color:black;
     font-family: Arial, Helvetica, sans-serif;
     font-size:1.15vw;
+    width: 175%;
   }
   
 
 </style>
+<script>
+  // List of sentences
+var _CONTENT = [ 
```

Comment:
it is VERY BAD PRACTICE to use `var`; always use `const` unless a variable needs to be mutable, then use `let`. BUT NEVER USE CONST!!!